systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaST.java

```diff
@@ -813,7 +813,7 @@ void testEntityOperatorWithoutTopicOperator(ExtensionContext extensionContext) {
 
     @ParallelNamespaceTest
     void testEntityOperatorWithoutUserOperator(ExtensionContext extensionContext) {
-        final String namespaceName = extensionContext.getStore(ExtensionContext.Namespace.GLOBAL).get(Constants.NAMESPACE_KEY).toString();
+        final String namespaceName = Environment.isNamespaceRbacScope() ? NAMESPACE : extensionContext.getStore(ExtensionContext.Namespace.GLOBAL).get(Constants.NAMESPACE_KEY).toString();
```

Comment:
Why this (and following) line is different? Shouldn't you use the same method for getting the namespace?

Reply:
Correct